scripts/utils.ts

```diff
@@ -193,15 +193,29 @@ export async function specifyComponent(
 
   if (matchedComponents.length > 1 || !matchedComponents[0]) {
     component = (
-      await prompts({
-        type: 'select',
-        name: 'component',
-        message: 'Select a component:',
-        choices: (matchedComponents.length > 1 ? matchedComponents : allComponents).map(name => ({
-          title: name,
-          value: name
-        }))
-      })
+      await prompts(
+        {
+          type: 'autocomplete',
+          name: 'component',
+          message: 'Select a component:',
+          choices: (matchedComponents.length > 1 ? matchedComponents : allComponents).map(name => ({
+            title: name,
+            value: name
+          })),
+          onState(this: any) {
+            this.fallback = { title: this.input, value: this.input }
+            if (this.suggestions.length === 0) {
+              this.value = this.fallback.value
+            }
+          }
+        },
+        {
+          onCancel: () => {
+            console.log('Canceled by user, exiting...')
+            process.exit(1)
+          }
+        }
```

Comment:
This `onCancel` is unnecessary, because there is an exit judgment below.

Reply:
Thanks, callback has been removed.